testing/test_deep_learning_pipeline.py

```diff
@@ -1,68 +1,33 @@
 import math
 
 import numpy as np
-from sklearn.cluster import KMeans
-from sklearn.datasets import load_boston
-from sklearn.decomposition import PCA, FastICA
-from sklearn.ensemble import GradientBoostingRegressor
-from sklearn.linear_model import Ridge
+from sklearn import linear_model
 from sklearn.metrics import mean_squared_error
-from sklearn.utils import shuffle
 
 from neuraxle.api import DeepLearningPipeline
-from neuraxle.hyperparams.distributions import RandInt, LogUniform, Boolean
-from neuraxle.hyperparams.space import HyperparameterSpace
+from neuraxle.metaopt.auto_ml import RandomSearch
+from neuraxle.metaopt.random import ValidationSplitWrapper
 from neuraxle.pipeline import Pipeline
-from neuraxle.steps.numpy import NumpyTranspose
-from neuraxle.steps.sklearn import SKLearnWrapper
-from neuraxle.union import AddFeatures, ModelStacking
+
+N_ITER = 1
+
+TIMESTEPS = 10
 
 VALIDATION_SIZE = 0.1
 BATCH_SIZE = 32
-N_EPOCHS = 10
+N_EPOCHS = 15
+
+DATA_INPUTS_PAST_SHAPE = (BATCH_SIZE, TIMESTEPS)
 
 
 def test_deep_learning_pipeline():
     # Given
-    boston = load_boston()
-    data_inputs, expected_outputs = shuffle(boston.data, boston.target, random_state=13)
-    expected_outputs = expected_outputs.astype(np.float32)
-    data_inputs = data_inputs.astype(np.float32)
-
-    pipeline = Pipeline([
-        AddFeatures([
-            SKLearnWrapper(
-                PCA(n_components=2),
-                HyperparameterSpace({"n_components": RandInt(1, 3)})
-            ),
-            SKLearnWrapper(
-                FastICA(n_components=2),
-                HyperparameterSpace({"n_components": RandInt(1, 3)})
-            ),
-        ]),
-        ModelStacking([
-            SKLearnWrapper(
-                GradientBoostingRegressor(),
-                HyperparameterSpace({
-                    "n_estimators": RandInt(50, 600), "max_depth": RandInt(1, 10),
-                    "learning_rate": LogUniform(0.07, 0.7)
-                })
-            ),
-            SKLearnWrapper(
-                KMeans(n_clusters=7),
-                HyperparameterSpace({"n_clusters": RandInt(5, 10)})
-            ),
-        ],
-            joiner=NumpyTranspose(),
-            judge=SKLearnWrapper(
-                Ridge(),
-                HyperparameterSpace({"alpha": LogUniform(0.7, 1.4), "fit_intercept": Boolean()})
-            ),
-        )
-    ])
+    data_inputs, expected_outputs = create_2d_data()
 
     p = DeepLearningPipeline(
-        pipeline,
+        Pipeline([
+            linear_model.LinearRegression()
+        ]),
```

Comment:
```
        Pipeline([
            linear_model.LinearRegression()
        ]),
```
==>
```
        linear_model.LinearRegression()
```
?

Reply:
done in demo j

Reply:
@alexbrillant then resolve all here please (if they are all resolved) and then close this PR.

Reply:
in this case, I really need a pipeline !! it wraps the linear regression with a SKLearnWrapper !!

Reply:
won't fix.